components/PostCardList/PostCardListItem.tsx

```diff
@@ -0,0 +1,227 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import Post from "../../models/Post";
+import calculateFinalPot from "../../utilities/calculateFinalPot";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+import {
+  getRelativeDateString,
+  getRelativeShortDateString
+} from "../../utilities/getRelativeDateString";
+import getStringWithSIMetricSuffix from "../../utilities/getStringWithSIMetricSuffix";
+import Card from "../Card";
+import { ThumbsUpIcon } from "../Icon";
+import PlayingCard from "../PlayingCard";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  post: Post;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItem({ post, ...props }: Props) {
+  const postType = React.useContext(PostTypeContext);
+
+  return (
+    <Root {...props}>
+      <HeroHand>
+        <HeroHandBackGround />
+        <HandCard
+          suit={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.left
+              .suit
+          }
+          rank={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.left
+              .rank
+          }
+        />
+        <HandCard
+          suit={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.right
+              .suit
+          }
+          rank={
+            post.gameSituation.playerCards[post.gameSituation.heroIndex]!.right
+              .rank
+          }
+        />
+      </HeroHand>
+
+      <PostTitle>{post.title}</PostTitle>
+
+      <Likes>
+        <ThumbsUpIcon />
+        {post.likes}
+      </Likes>
+
+      <Attributes>
+        <Attribute>
+          <h5>Play at</h5>
```

Comment:
I don't get why this is `h5`. Typically people use **headings** from `h1` to `h6` in order. For instance, the most important heading is `h1`, and then the second one would be `h2`. However, there's no `h1` to `h4` but suddenly `h5` appears.